src/main/java/com/jcabi/w3c/BaseValidator.java

```diff
@@ -224,7 +179,7 @@ private Defect defect(final XML node) {
      * @return The value
      * @see #intOf(List)
      */
-    private static String textOf(final List<String> lines) {
+    protected static String textOf(final List<String> lines) {
```

Comment:
@carlosmiranda I've got error here: [ERROR] \src\main\java\com\jcabi\w3c\BaseValidator.java[182-190]: Use explicit scoping instead of the default package private level (DefaultPackage) - this is why I choose "protected"